handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -148,7 +168,7 @@ func claimAsString(t *jwt.Token, claim string) (string, bool) {
 	return "", false
 }
 
-func claimAsInt64(t *jwt.Token, claim string) (int64, bool) {
+func ClaimAsInt64(t *jwt.Token, claim string) (int64, bool) {
```

Comment:
I don't see this function used outside of this package, so maybe it shouldn't be exported?